### PR TITLE
Add the video URL to the proposal page 

### PR DESCRIPTION
**Pull request by sayanchowdhury**
This PR adds the feature to update the proposal with the video URL. The
video URL will be containing 1-2 min video about the talk from the speaker.

This would help to promote the talk as well as judging better on how the speaker
is.

### CODE REVIEW

**Comment by palnabarun** [comment]
Note: nit: The `meta` commit needs to go be removed from this branch.

File: junction/proposals/forms.py [hunk 1]
```diff
@@ -114,6 +114,11 @@ class ProposalForm(forms.Form):
         required=False,
         help_text="What should the participants know before attending your session?",
     )
+    video_url = forms.CharField(
+        required=False,
+        help_text="Short 1-2 min video describing your talk",
+        widget=forms.TextInput(attrs={"class": "charfield"}),
```
Comment (palnabarun) [inline junction/proposals/forms.py:120]: Can you make the `help_text` on both the form and model the same?

File: junction/proposals/migrations/0027_auto_20200502_0540.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        migrations.AddField(
+            model_name="proposal",
+            name="video_url",
+            field=models.URLField(
+                blank=True,
+                default="",
+                help_text="A short 1-2 mins video link about your talk",
+            ),
+        ),
+    ]
```
Comment (palnabarun) [inline junction/proposals/migrations/0027_auto_20200502_0540.py:33]: Just trying to understand why there are two migrations?

Comment (sayanchowdhury) [reply junction/proposals/migrations/0027_auto_20200502_0540.py:33]: Yes, these changes are tested. The historical data is added from `django-simple-history` which I guess provides kind of an audit log.

image

**Pull request merged**